pkgs/applications/kde/kdenlive.nix

```diff
@@ -65,7 +65,7 @@ mkDerivation {
     qtwebkit
     shared-mime-info
     libv4l
-    ffmpeg-full
+    ffmpeg
```

Comment:
Could we link to an issue saying ffmpeg-full is broken?

Reply:
Fixed separately